### PR TITLE
Sending company name and number with transaction

### PR DESCRIPTION
**Pull request by abhiahirv1**
__Short description outlining key changes/additions__

__JIRA Ticket Number__

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/java/uk/gov/companieshouse/acsp/service/AcspService.java [hunk 1]
```diff
@@ -98,6 +98,14 @@ public ResponseEntity<Object> updateACSPDetails(Transaction transaction,
             throw new InvalidTransactionStatusException(String.format(
                     "Can't update transaction with stastus: %s ", transaction.getStatus().toString()));
         }
+
+        if (acspDataDto.getCompanyDetails() != null) {
```
Comment (ttingle-ch) [inline src/main/java/uk/gov/companieshouse/acsp/service/AcspService.java:102]: What about unincorporated and Sole trader scenario, the company name is not saved in companyDetails for this

**Review by sardhani-ch — APPROVED** [review]

**Review by sampankumar — DISMISSED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Pull request merged**